lib/resources/r4/observation.yaml

```diff
@@ -193,7 +193,18 @@ fields:
   required: 'No'
   type: performerFunction
   description: Distinguishes the type of involvement of the performer in the event.
-  action: create
+  action:
+    - terminology
+    - create
+  binding:
+    description: Codes identifying the function of the performer.
+    terminology:
+    - display: Performer function codes
+      system: http://terminology.hl7.org/CodeSystem/v3-ParticipationType
+      info_link: http://hl7.org/fhir/R4/valueset-performer-function.html
+    - display: Participation codes
```

Comment:
I would like Fran's thought on display names for this links

Reply:
I think that was the name in the code set when you open that system link below. I'm okay with Participation Codes.